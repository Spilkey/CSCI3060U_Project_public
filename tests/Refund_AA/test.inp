login
user03
login
user04
login
user01
refund
user03
user04
100
logout
logout
logout