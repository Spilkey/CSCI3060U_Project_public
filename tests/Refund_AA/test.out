05 user03          user04          000000100
00 user01          AA 000000.00
00 user03          BS 0000000.00
00 user04          SS 000000.00